.mailmap

```diff
@@ -971,7 +971,7 @@ Noumbissi valere Gille Geovan <noumbissivalere@gmail.com>
 Oleksandr Gituliar <gituliar@gmail.com>
 Oliver Lee <oliverzlee@gmail.com>
 Omar Wagih <o.wagih.ow@gmail.com> OmarWagih1 <o.wagih.ow@gmail.com>
-Omkaar <79257339+Pysics@users.noreply.github.com>
+Omkaar <79257339+Infiniticity@users.noreply.github.com>
```

Comment:
The original commits are already stored under the previous email address. Rather than just changing it you need to make an alias like:
```
Omkaar <79257339+Infiniticity@users.noreply.github.com> Omkaar <79257339+Pysics@users.noreply.github.com>
```
Also do you want your name to just be recorded as `Omkaar` or would you prefer to have your full name (like most other names in this file)?

Reply:
Just `Omkaar` is fine.